packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -265,7 +265,16 @@ class AssetServer {
   Directory partFiles;
 
   Future<Response> handle(Request request) async {
-    if (request.url.path.contains('stack_trace_mapper')) {
+    if (request.url.path.endsWith('.html')) {
```

Comment:
This is a complicated file handler.

Shelf also has static handlers that can serve all files out of a directory, which might make this logic a little simpler.

Reply:
Agreed, but as a long term goal I would like to remove our `shelf` dependency